styles/styles.css

```diff
@@ -0,0 +1,278 @@
+* {box-sizing: border-box;}
+
+button, input, select, textarea {
+  font-family: inherit;
+  font-size: 100%;
+}
+.skip-hidden {
+  position: absolute;
+  top: -10000px;
+  left: -10000px;
+  height: 1px;
+  width: 1px;
+  text-align: left;
+  overflow: hidden;
+}
+a.skip-hidden:focus {
+  left: 0;
+  top: 0;
+  width: auto;
+  height: auto;
+  overflow: visible;
+}
+header {
+  display: flex;
+  flex-direction: row;
+  background-image: url(header-image.jpeg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+  background-color: #0F0C0C;
+  border: solid .1rem;
+  width: 85%;
+  min-width: 320px;
+  max-width: 1200px;
+  height: 25rem;
+}
+.logo-container {
+  margin-left: 5rem;
+  margin-top: 2.5rem;
+}
+h1 {
+  color: #32CD32;
+  font-size: 1.5rem;
+}
+.cart-price {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap-reverse;
+  color: white;
+  margin-left: 33rem;
```

Comment:
this margin left is awkward. Perhaps you could play with justify-content on the parent container to make things appear to the end of the flex container, rather than hammering it over with margin.